scripts/build_data.py

```
@@ -7774,7 +7581,11 @@ def should_replace_raw_search_candidates_with_locality_bias(
 )
 PLACE_PAGE_ADDRESS_REJECT_HOST_FRAGMENTS = ("gstatic.com", "googleusercontent.com")
 PLACE_PAGE_ADDRESS_ENTITY_TOKEN_RE = re.compile(r"^/(?:g|m)/[A-Za-z0-9_-]+$")
-PLACE_PAGE_URL_LIKE_RE = re.compile(r"(?:https?://|www\.)", re.IGNORECASE)
+PLACE_PAGE_URL_LIKE_RE = re.compile(
+    r"(?:https?://|www\.|/(?:search|maps|url|local)(?:[/?#]|$))",
+    re.IGNORECASE,
+)
+PLACE_PAGE_TRAVEL_PRODUCT_ADDRESS_RE = re.compile(r"\b\d+\s*[-–]\s*day\b", re.IGNORECASE)
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)